WTraining/app/models/user.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const bcrypt = require('bcryptjs');
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
```

Comment:
Remember that the models should be defined with the first letter in capital letter